solutions/USACO jan19-icy-perimeter.mdx

```diff
@@ -0,0 +1,93 @@
+---
+id: usaco-895
+source: USACO Silver 2019 January
+title: Icy Perimeter
+author: Tanish Tyagi 
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_perimeter_silver_jan19.html)
+
+<LanguageSection>
+
+<CPPSection>
+
+## C++ Implementation
+
+```cpp
+//created by Tanish Tyagi
+#include <bits/stdc++.h>
+
+using namespace std;
+using ll = long long;
+
+const int MAXN = 1005;
+
+char grid[MAXN][MAXN]; // the grid itself
+int n;
+bool visited[MAXN][MAXN]; // keeps track of which nodes have been visited
+ll currA = 0;
```

Comment:
`ll curA = 0; // Keeps track of the current area of the current blob of ice cream'.

Reply:
pls address this